core/templates/services/UpgradedServices.ts

```diff
@@ -341,6 +341,10 @@ import { ProfilePageBackendApiService } from
 import { PythonProgramTokenizer } from 'classifiers/python-program.tokenizer';
 import { QuestionBackendApiService } from
   'domain/question/question-backend-api.service.ts';
+import { QuestionValidationService } from
+  'services/question-validation.service.ts';
+import { QuestionObjectFactory } from
+  'domain/question/QuestionObjectFactory';
```

Comment:
Additions to UpgradedServices are not required anymore. Can you remove these additions?

Reply:
Fixed!